### ISSUE TITLE
Add angular-1.x template

### ISSUE THREAD

**Issue by MarshallOfSound** [electron-userland/electron-forge-templates#3]
The README mentions as a supported framework so it would be good to get a super basic example in here

**Comment by adriatic** [comment]
I am interested in creating the Aurelia template for electron forge. As ut seems you do not have any documentation describing the process of creating a new template - and based on the date of this issue creation, I would guess that there is no interest in making Angular 1. x template.

I am puzzled why is that? The electron forge is a great idea, something GitHub / Electron team should have done long time ago. Based on @MarshallOfSound's success with Google player, there is no shortage of talent at **[userland](https://github.com/electron-userland)**, there is the much needed **[link to GitHub](https://github.com/zeke)**  ... so whay are things not moving faster?

I am mostly interested in developing tools using Aurelia, but given the fact that Aurelia is also short on development resources I end up working on tools to improve the Aurelia user's development experience. This is the reason for finding this (electron-forge-templates) project.

**Comment by adriatic** [comment]
Just realized that there is an action item asking for **[Aurelia template](https://github.com/electron-userland/electron-forge-templates/issues/44)** - so the above comments should be placed there. Now, they will be ___referenced___ there.

**Comment by malept** [comment]
> so whay are things not moving faster?

The reason is because Electron Forge (and really, everything under the Electron Userland org) is volunteer driven. If no one has the time or inclination to work on something, it doesn't get worked on. These issues are labeled "Help Wanted" for a reason - they're waiting for a member of the community to claim and work on it.

**Comment by JustinBeaudry** [comment]
I've implemented a basic angular1 template, but I'm curious what some thoughts are here. I would like to keep the template as simple as possible, but I would also like to remove boilerplate as well (which is the intention).

Here's what I've implemented so far:

- basic template implementation
- `eslint` support
  - `eslint-plugin-angular`
- `electron-devtools-installer` support
  - `ANGULARJS_BATARANG`

Here are my outstanding questions:

~~1. Babel support would be a nice-to-have but doesn't feel necessary (could be a separate template akin to `react-typescript`, e.g. `angular1-es6`~~ ~~(forgoing this for now as it doesn't feel necessary)~~ (actually read through the code and realized this is already implemented 🤦‍♂️)
~~2. Router support baked in (ui-router)~~ (*angular2 template does not include routing, so I'm going to assume that this is also a nice-to-have, but not necessary*)

*UPDATE:  added PR #61*

cc @MarshallOfSound
